pages/users/[id]/bids/placed.tsx

```diff
@@ -77,16 +71,13 @@ const BidPlacedPage: NextPage<Props> = ({ now }) => {
     },
   })
 
-  const bidData = useMemo(() => data || previousData, [data, previousData])
-  const loading = _loading && !bidData
-
   const bids = useMemo(
     () =>
-      (bidData?.bids?.nodes || []).map((x) => ({
+      (data?.bids?.nodes || []).map((x) => ({
```

Comment:
The fallback to empty array could be removed so `bids` would also be undefined when `data?.bids?.nodes` is undefined.
```suggestion
      data?.bids?.nodes.map((x) => ({
```
This will allow to update the logic line 178 - 180 to use `bids` and be safer:
```tsx
          {bids === undefined ? (
            <Loader />
          ) : bids.nodes.length > 0 ? (
```

Reply:
[simplify array checks](https://github.com/liteflow-labs/starter-kit/pull/367/commits/f5700bc8ca546c8ee17e25ce2fcebf436e8de13c)
Because of this one, data check on useMemo wasn't triggering loading state, so had to also push this commit which makes sense.
[useMemo dependency change to show loader correctly](https://github.com/liteflow-labs/starter-kit/pull/367/commits/e84ff65447de557f8caf36f070a3b17eb3c31437)